src/NuGet.Clients/PackageManagement.UI/Converters/InverseBooleanConverter.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Globalization;
+using System.Windows.Data;
+
+namespace NuGet.PackageManagement.UI
+{
+    public class InverseBooleanConverter : IValueConverter
```

Comment:
I'd be inclined to write a standard boolean converter and invert it, which would also be consistent with NuGet.PackageManagement.UI.BooleanToVisibilityConverter.